crawler.c

```diff
@@ -268,17 +268,20 @@ static void crawler_metadump_eval(crawler_module_t *cm, item *it, uint32_t hv, i
         refcount_decr(it);
         return;
     }
+    client_flags_t flags;
+    FLAGS_CONV(it, flags);
     // TODO: uriencode directly into the buffer.
     uriencode(ITEM_key(it), keybuf, it->nkey, KEY_MAX_URI_ENCODED_LENGTH);
     int total = snprintf(cm->c.buf + cm->c.bufused, 4096,
-            "key=%s exp=%ld la=%llu cas=%llu fetch=%s cls=%u size=%lu\n",
+            "key=%s exp=%ld la=%llu cas=%llu fetch=%s cls=%u size=%lu flags=" CLIENT_FLAG_FORMAT_SPECIFIER "\n",
```

Comment:
Value is an unsigned int, so you can just do `%llu` here and drop the macro. I do this in a few other specific places already.

Reply:
Thanks for the feedback @dormando! Fixed.